cmake/modules/SetUpWindows.cmake

```
@@ -47,6 +47,9 @@ elseif(MSVC)
     set(CMAKE_C_FLAGS   "-nologo -I${CMAKE_SOURCE_DIR}/build/win -FIw32pragma.h -FIsehmap.h ${BLDCFLAGS} -EHsc- -W3 -D_WIN32 -DNOMINMAX")
     install(FILES ${CMAKE_SOURCE_DIR}/build/win/w32pragma.h  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR} COMPONENT headers)
     install(FILES ${CMAKE_SOURCE_DIR}/build/win/sehmap.h  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR} COMPONENT headers)
+    if(win_broken_tests)
+      set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_ENABLE_BROKEN_TESTS")
```

```suggestion
      set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DR__ENABLE_BROKEN_WIN_TESTS")
```
Rationale:
- the test isn't broken, except on Windows.
- starting with `_` is evil; let's stick to usual ROOT naming convention for CPP macros and call it `R__...`